mk/deploy.mk

```diff
@@ -98,32 +89,27 @@ s3copybranch: guard-DEPLOY_TARGET \
               .build-artefacts/requirements.timestamp
 	PROJECT=${PROJECT} ${PYTHON_CMD} ./scripts/s3manage.py upload --force --url $(S3_BUCKET_URL) ${CODE_DIR} ${DEPLOY_TARGET} ${NAMED_BRANCH} ${DEPLOY_GIT_BRANCH};
 
-s3list := $(patsubst %,s3list%,int,infra,prod)
+s3list := $(patsubst %,s3list%,int,prod)
 PHONY: $(s3list)
 s3list%: .build-artefacts/requirements.timestamp
 	${PYTHON_CMD} ./scripts/s3manage.py list $(S3_BUCKET_$(shell echo $*| tr a-z A-Z))
 
-s3info := $(patsubst %,s3info%,int,infra,prod)
+s3info := $(patsubst %,s3info%,int,prod)
 PHONY: $(s3info)
 s3info%: guard-S3_VERSION_PATH .build-artefacts/requirements.timestamp
 	${PYTHON_CMD} ./scripts/s3manage.py info ${S3_VERSION_PATH} $(S3_BUCKET_$(shell echo $*| tr a-z A-Z));
 
-s3activate := $(patsubst %,s3activate%,int,infra,prod)
+s3activate := $(patsubst %,s3activate%,dev,int,prod)
 PHONY: $(s3activate)
 s3activate%: guard-DEPLOY_GIT_BRANCH \
 		         guard-SNAPSHOT \
-	guard-VERSION .build-artefacts/requirements.timestamp
-	${AWS_CMD} s3 cp $(S3_OPTS) --recursive s3://$(S3_BUCKET_$(shell echo $*| tr a-z A-Z))/${DEPLOY_GIT_BRANCH}/${SNAPSHOT}/  s3://$(S3_BUCKET_$(shell echo $*| tr a-z A-Z))/
-
-s3activatedist := $(patsubst %,s3activatedist%,dev,infra,prod)
-PHONY: $(s3activatedist)
-s3activatedist%: guard-DEPLOY_GIT_BRANCH \
-		         guard-SNAPSHOT \
-						 guard-S3_BUCKET_INT \
-	guard-VERSION .build-artefacts/requirements.timestamp
+		         guard-S3_BUCKET_INT \
+		         guard-S3_BUCKET_PROD \
+		         guard-VERSION \
+		         .build-artefacts/requirements.timestamp
 	${PYTHON_CMD} ./scripts/s3manage.py activate --branch ${DEPLOY_GIT_BRANCH} --version ${VERSION} $(DEPLOY_TARGET);
```

Comment:
`Makefile` choose the right project and hence the right bucket to use. And `s3manage`do the same.